server/schemas/mutations/deleteJob.js

```diff
@@ -0,0 +1,13 @@
+const { Job } = require('../../models');
+
+
+async function deleteJob (parent, { jobId }) {
+    if (context.user) {
+        const job = await Job.findByIdAndDelete({
+            _id: jobId,
+        });
+    return job     
+    }
```

Comment:
Since this will delete the job, not sure what this will return. Trying to figure out with testing what this will return.